pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTest.java

```diff
@@ -348,7 +348,7 @@ protected Map<String, String> getStreamConfigMap() {
       streamConfigMap.put(KafkaStreamConfigProperties.constructStreamProperty(
           KafkaStreamConfigProperties.HighLevelConsumer.KAFKA_HLC_ZK_CONNECTION_STRING), getKafkaZKAddress());
       streamConfigMap.put(KafkaStreamConfigProperties.constructStreamProperty(
-              KafkaStreamConfigProperties.HighLevelConsumer.KAFKA_HLC_BOOTSTRAP_SERVER),
+          KafkaStreamConfigProperties.HighLevelConsumer.KAFKA_HLC_BOOTSTRAP_SERVER),
```

Comment:
curious, do you use the right formatter?

Reply:
@yupeng9 I re-imported but the codestyle seems has slight difference between each intellij versions.